src/features/MoviesBrowser/index.js

```diff
@@ -1,3 +1,7 @@
-const MoviesBrowser = () => <></>;
+import MovieTile from "../../common/MovieTile";
+
+const MoviesBrowser = () => <>
+    <MovieTile />
+</>;
```

Comment:
Using MovieTile like this is an option in order to avoid doing it manually by reviewers, but I think everyone is able to check it in their own private environment without publishing it, so it's not really necessary 👌

Reply:
Yesterday we decided that we will do it that way and you agreed :D But as I said - I think, that from now on, there will be no need for that kind of activity ;)

Reply:
I just said that's not really necessary same as here, but I'm not really against it (we will have to remove it anyways later on) 👌